src/current/cockroachcloud/regions.md

```diff
@@ -22,11 +22,15 @@ For optimal performance, configure your cluster's regions to be as near as possi
 {{site.data.alerts.callout_info}}
 Creating a CockroachDB {{ site.data.products.serverless }} cluster on Azure is not supported.
 {{site.data.alerts.end}}
+</section>
 
 ## AWS regions
 
+<section class="filter-content" markdown="1" data-scope="serverless">
```

Comment:
in future it would be good to do these type of refactors in a separate PR

this PR is adding a few new regions, and it would be much easier to review if the diff were just "6 new regions = 6 line diff"

Reply:
True. I made it a separate commit to try to help with that. The refactor happened because I completely forgot there were tabs and got very confused doing the edit initially.